lib/AST/ASTContext.cpp

```diff
@@ -1683,7 +1683,7 @@ namespace {
       return OrderDeclarations(SrcMgr)(lhs, rhs);
     }
   };
-}
+} // namespace
```

Comment:
@jrose-apple I've now changed all instances of `// namespace` to `// end anonymous namespace`.